ank/src/cli_commands/get_agents.rs

```diff
@@ -49,18 +50,30 @@ impl CliCommands {
 }
 
 fn transform_into_table_rows(
-    agents_map: impl Iterator<Item = String>,
+    agents_map: impl Iterator<Item = (String, FilteredAgentAttributes)>,
     workload_states_map: &WorkloadStatesMap,
 ) -> Vec<AgentTableRow> {
     let mut agent_table_rows: Vec<AgentTableRow> = agents_map
-        .map(|agent_name| {
+        .map(|(agent_name, agent_attributes)| {
             let workload_states_count = workload_states_map
                 .get_workload_state_for_agent(&agent_name)
                 .len() as u32;
 
             AgentTableRow {
                 agent_name,
                 workloads: workload_states_count,
+                cpu_usage: agent_attributes
+                    .agent_resources
+                    .clone()
+                    .unwrap()
+                    .cpu_usage
+                    .unwrap(),
+                free_memory: (agent_attributes
+                    .agent_resources
+                    .unwrap()
+                    .free_memory
+                    .unwrap()
+                    / 100) as f32,
```

Comment:
These unwraps must be removed.

Reply:
The free memory is being expressed as a float (percentage) in the sysinfo crate, but due to some internal restrictions it has to be sent as a u32, so the only place where it is relevant to view it as a percentage is in the cli.

Reply:
Updated this in order to show empty strings if no available cpu_load or free_memory is available. Still need to find a solution on how to correctly show the cpu_load as floating point percent

Reply:
Unwraps still there. There is an extra comment for this now